README.adoc

```diff
@@ -0,0 +1,161 @@
+{nbsp} +
+
+[[chatui-navigation-guide]]
+The UI is designed to model a chat room where multiple users can connect/disconnect and exchanging chat messages. You simulate the chat room activities in one single browser window. This makes it easy to see the real time interactions, server-side pushes etc, without needing to open many browser tabs. Each user is a separate iframe with a separate websocket connection. The screen below shows the possible actions to interact with the UI.
+
+image::docs/images/ChatUI_NavigationGuide.png[Chat Application - Navigation Guide, align="center"]
+{nbsp} +
+
+== Code implementation
+
+[horizontal]
+chat-ui:: directory containing the code of the stateless service `chat`. This code is written in TypeScript, and run as a Node.js server.
+deploy:: directory containing the yaml files to deploy all the services composing the Chat application. The target cluster could be your own Kubernetes cluster or https://docs.lbcs.dev/index.html[Lightbend Cloudstate managed service]
+
+The code of the backend services `Friends` and `Presence` is separate from the UI code. This is intentional to demonstrate the polyglot aspect of Cloudstate services. They can seamlessly interact regardless of the programming language used to implement the service. Below are the available implementations. You can choose any combination for deployment. For example, `Friends` service in Javascript and `Presence` service in Java.
+
+=== JavaScript implementation
+
+* https://github.com/cloudstateio/samples-js-chat/tree/master/friends[Friends service]
+* https://github.com/cloudstateio/samples-js-chat/tree/master/presence[Presence service]
+
+=== Java implementation
+
+* https://github.com/cloudstateio/samples-java-chat/tree/master/friends[Friends service]
+* https://github.com/cloudstateio/samples-java-chat/tree/master/presence[Presence service]
+
+
+== Prerequisites
+
+=== Javascript Dev environment
+
+https://github.com/cloudstateio/samples-js-chat/blob/master/README.adoc#js-devenv-setup-for-cloudstate[Javascript Dev environment setup for Cloudstate]
+
+=== Infrastructure
+
+* A Docker repository that the Kubernetes cluster has the necessary credentials to pull from. You can get a free Docker registry at https://hub.docker.com/
+
+For the deployment, there are two possible choices. Below we highlight the main characteristic of each deployment approach. Further details about prerequisites and infrastructure preparation would be discussed further in the <<deployment,Deployment>> section. 
+
+==== Deploy on Lighbend Cloudstate managed service
+
+Basically, all you need is to open an account with Lightbend Cloudstate hosting service. Then install the "Cloudstate CLI" tool and you are good to deploy. There is no need to provide any local infrastructure.
+
+The deployment is as simple as applying the yaml configuration files. Lightbend Cloudstate managed service will take care of the hosting, configuration and maintenance of your service. 
```

Comment:
I think there are no yaml files any longer, everything is done through the UI or the csctl utility. Maybe you could just link to the instructions from here.

Reply:
Oh good point. Thx. WIll update accordingly